frontend/schema/class-schema-website.php

```diff
@@ -54,6 +54,10 @@ public function generate() {
 			'name'      => $this->context->site_name,
 		);
 
+		if ( get_bloginfo( 'description' ) ) {
+			$data['description'] = $this->context->description;
```

Comment:
Turns out `$this->context->description` is the single post meta description. Instead, we need the site tagline. I guess a new `$site_description` property should be set in `WPSEO_Schema_Context`.